transport/spi/src/test/java/org/kaazing/gateway/transport/LoggingFilterTest.java

```diff
@@ -38,8 +38,6 @@
 import org.jmock.lib.legacy.ClassImposteriser;
 import org.junit.Rule;
 import org.junit.Test;
-import org.slf4j.Logger;
-
 import org.kaazing.gateway.resource.address.IdentityResolver;
 import org.kaazing.gateway.resource.address.ResourceAddress;
 import org.kaazing.gateway.resource.address.ResourceAddressFactories;
```

Comment:
Can you delete ResourceAddressFactories and let the test use ResourceAddressFactory.